nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Good job so far by implementing the nameable class, but I think it would be better if you return a message that for example, "decorator has not implemented the method correct name" 🙃. The line can be similar to:

```ruby
   raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
```